doc/get_started.rst

```diff
@@ -3,110 +3,89 @@
 Get started
 ===========
 
+Installation
+------------
 
-Install
--------
-
-This package can be installed through `pip`.  In order to allow benchopt to automatically
-install solvers dependencies, the install needs to be done in a `conda` environment.
+The recommended way to use benchopt is within a conda environment to fully-benefit from all its features.
+Hence, start by creating a dedicated conda environment and then activate it.
 
 .. prompt:: bash $
 
     conda create -n benchopt python
     conda activate benchopt
 
-To get the **latest release**, use:
+Benchopt is available on PyPi. You can get the **stable version** via ``pip`` by running the following command
 
 .. prompt:: bash $
 
-    pip install benchopt
+    pip install -U benchopt
 
-And to get the **latest development version**, you can use:
+Eager to try out the **development version**? you can run instead
 
 .. prompt:: bash $
 
-    pip install -U -i https://test.pypi.org/simple/ benchopt
-
-This will install the command line tool to run the benchmark. Then, existing
-benchmarks can be retrieved from GitHub or created locally. To discover which
-benchmarks are presently available look for
-`benchmark_* repositories on GitHub <https://github.com/benchopt/>`_,
-such as for `Lasso -- l1-regularized linear regression <https://github.com/benchopt/benchmark_lasso>`_.
-This benchmark can be retrieved locally with:
+    pip install -U -i https://test.pypi.org/simple/benchopt
 
-.. prompt:: bash $
+.. attention::
 
-    git clone https://github.com/benchopt/benchmark_lasso.git
+   The **development version** is a work in progress and hence might contain incomplete features.
+   A typical user is advised to use the **stable version** instead.
 
+With benchopt being installed, you get access to the :ref:`Command Line Interface (CLI) <cli_ref>`,
+which enables simple and easy manipulation of benchmarks just from the terminal.
```

Comment:
```suggestion
which enables running benchmarks from the terminal.
```

Reply:
The CLI is not only for running benchmarks, it is also for installation, publishing ...

I would keep
```suggestion
which enables a simple manipulation of benchmarks from the terminal.
```

Reply:
I'm -1 with removing this part. It is important to hint that the proper way to use `benchopt` is through the CLI.

As a side note, getting feedback from one user is to be considered not to be directly implemented. It is tricky to please everyone.

Reply:
yes, that's why I asked to remove the "running a benchmark through a python file" :) all the doc uses the benchopt cli tool
readding a removed sentence on the benchopt CLI :)